### PR TITLE
Tranliteration link admin

### PR DESCRIPTION
**Pull request by Tysyatsky**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @Kasterov 
- [ ] @Katerix 
- [ ] @Kotusyk 
- [ ] @MementoMorj 
- [ ] @NadiaKishchuk 

## Summary of issue

Transliteration in the streetcode url needed to be implemented. Frontend part.

## Summary of change

Added transliteration link in the url.

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/app/common/hooks/stateful/useRouter.hook.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { useLocation, useParams } from 'react-router-dom';
 
 export const useRouteId = () => {
     const params = useParams<{ id: string }>();
-    return parseInt(params.id ?? '1', 10);
+    return params.id ?? '404';
```
Comment (NadiaKishchuk) [inline src/app/common/hooks/stateful/useRouter.hook.ts:6]: I think it makes sense to name this method as useRouteLink or something like this for better code readability

File: src/features/StreetcodePage/MapBlock/MapBlock.component.tsx [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
+import MapOSM from './Map/Map.component';
 
 const MapBlock = () => {
-    const id = useRouteId();
+    const { streetcodeStore: { getStreetCodeId } } = useMobx();
     const toponyms = useAsync(() => ToponymsApi
         .getAll(), []).value as Toponym[];
     const streetcodeCoordinates = useAsync(() => StreetcodeCoordinatesApi
-        .getByStreetcodeId(id), [id]).value as StreetcodeCoordinate[];
+        .getByStreetcodeId(getStreetCodeId ?? 1), [getStreetCodeId]).value as StreetcodeCoordinate[];
```
Comment (NadiaKishchuk) [inline src/features/StreetcodePage/MapBlock/MapBlock.component.tsx:22]: Are you sure that it is good to get the value with streetcode id 1 if the store doesn`t have the streetcode id?

File: src/features/StreetcodePage/PartnersBlock/Partners.component.tsx [hunk 3]
```diff
@@ -8,14 +8,14 @@ import useMobx from '@stores/root-store';
...
-    const { partnersStore } = useMobx();
+    const { partnersStore, streetcodeStore: { getStreetCodeId } } = useMobx();
     const { fetchPartnersByStreetcodeId, getPartnerArray } = partnersStore;
 
     const streetcodeId = useRouteId();
 
     useAsync(
         () => Promise.all([
-            fetchPartnersByStreetcodeId(streetcodeId),
+            fetchPartnersByStreetcodeId(getStreetCodeId ?? 1),
```
Comment (NadiaKishchuk) [inline src/features/StreetcodePage/PartnersBlock/Partners.component.tsx:18]: the same

**Review by NadiaKishchuk — APPROVED** [review]

File: src/app/common/hooks/stateful/useRouter.hook.ts (hunk 1 above)
Comment (Tysyatsky) [reply src/app/common/hooks/stateful/useRouter.hook.ts:6]: renamed to useRouteUrl)

**Pull request merged**